cypress/integration/publication_references_book.spec.js

```diff
@@ -17,9 +17,11 @@ describe('Publication: References: Book', () => {
     cy.get('[data-testid=publication-context-type]').click({ force: true }).type(' '); //makes the select options open
```

Comment:
en liten bug:
1. velg hovedtype
2. gå til "description"
3. gå til "reference"
4. velg subtype
5. feltet for tidsskrift har error state, selv brukeren ikke har kunne satt verdi enda

Feltet settes sikkert som touched på unmount

Reply:
ooops! will fix

Reply:
skal være fikset nå!

Reply:
hadde det ikke holdt å fikse `touchedReferenceTabFields` i `formik-helpers`? siden alt der settes til touched når man navigerer bort fra reference tab: https://github.com/BIBSYSDEV/NVA-Frontend/blob/develop/src/pages/publication/ReferencesPanel.tsx#L51

nå går vi sikkert helt bort fra å hardkode det slik når ny modell er på plass da, så det er kanskje ikke så nøye heller

Reply:
hehe, jo - det hadde det sikkert 😅  ja - vi må endre det etterhvert uansett